doctr/datasets/datasets/base.py

```diff
@@ -49,6 +53,18 @@ def __getitem__(
             # typing issue cf. https://github.com/python/mypy/issues/5485
             img = self.sample_transforms(img)  # type: ignore[call-arg]
 
+        if isinstance(target, dict) and 'boxes' in target.keys() and self.convert_to_relative:
+            # Convert to relative coordinates if not already
+            if not np.all((target['boxes'][:, :4] <= 1) & (target['boxes'][:, :4] >= 0)):
```

Comment:
Same here, this can removed, we'll send to correct flag to specify whether the data need conversion in getitem directly.